Wox/ViewModel/MainViewModel.cs

```diff
@@ -211,6 +214,46 @@ public string QueryText
             }
         }
 
+        private string _queryTextSuggestion;
+        public string QueryTextSuggestion
+        {
+            get
+            {
+                return QueryTextSuggestionFromResultOrEmpty();
+            }
+            set
+            {
+                _queryTextSuggestion = value;
+            }
+        }
+
+        private void HandleSelectedResultChange(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName != "SelectedItem")
```

Comment:
better to use nameof(Results.SelectedItem)

Reply:
Yes good point